### PR TITLE
rename test methods so adhere to coding standards

### PR DESCRIPTION
**Pull request by MarkRobinsonDfe**
Just renaming all the page test method names so they are the same / adhere to coding standards

### CODE REVIEW

File: Dfe.Academies.External.Web.UnitTest/Pages/Trust/ApplicationSelectTrustModelTests.cs [hunk 1]
```diff
@@ -15,11 +15,8 @@ namespace Dfe.Academies.External.Web.UnitTest.Pages.Trust;
 [Parallelizable(ParallelScope.All)]
 public class ApplicationSelectTrustModelTests
 {
-    // TODO MR:- OnGetAsync() / OnPostAsync()
-
-
     [Test]
-    public async Task ApplicationSelectTrustModel___TestSelectedUrnProperty__Valid()
+    public async Task TestSelectedUrnProperty__Valid()
```
Comment (CathLass) [inline Dfe.Academies.External.Web.UnitTest/Pages/Trust/ApplicationSelectTrustModelTests.cs:19]: same as comment for School page

File: Dfe.Academies.External.Web.UnitTest/Pages/Trust/ApplicationSelectTrustModelTests.cs [hunk 2]
```diff
@@ -42,7 +39,7 @@ public async Task ApplicationSelectTrustModel___TestSelectedUrnProperty__Valid()
     }
 
     [Test]
-    public async Task ApplicationSelectTrustModel___TestSelectedUrnProperty__InValid()
+    public async Task TestSelectedUrnProperty__InValid()
```
Comment (CathLass) [inline Dfe.Academies.External.Web.UnitTest/Pages/Trust/ApplicationSelectTrustModelTests.cs:42]: just spotted - invalid is all one word, so 'Invalid' not 'InValid'

File: Dfe.Academies.External.Web.UnitTest/Pages/WhatAreYouApplyingToDoModelTests.cs [hunk 3]
```diff
@@ -55,7 +55,7 @@ public async Task WhatAreYouApplyingToDoModel___OnGetAsync___InValid()
     }
 
     [Test]
-    public async Task WhatAreYouApplyingToDoModel___OnPostAsync___ModelIsValid___InValid()
+    public async Task OnPostAsync___ModelIsValid___InValid()
```
Comment (CathLass) [inline Dfe.Academies.External.Web.UnitTest/Pages/WhatAreYouApplyingToDoModelTests.cs:58]: I don't understand this test from the title and with a conditional assert it's hard to make out what should be happening.

**Review by CathLass — CHANGES_REQUESTED** [review]
Most comments are not musts - but changing all cases of 'InValid' to 'Invalid' worth doing?

File: Dfe.Academies.External.Web.UnitTest/Pages/School/ApplicationSelectSchoolModelTests.cs [hunk 4]
```diff
@@ -67,7 +71,7 @@ public async Task ApplicationSelectSchoolModel___TestSelectedUrnProperty__Valid(
     }
 
     [Test]
-    public async Task ApplicationSelectSchoolModel___TestSelectedUrnProperty__InValid()
+    public async Task TestSelectedUrnProperty__InValid()
```
[the remark this answers is not in the document]
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web.UnitTest/Pages/School/ApplicationSelectSchoolModelTests.cs]: renamed all the InValid -> Invalid

[the remark this answers is not in the document]
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web.UnitTest/Pages/School/ApplicationSelectSchoolModelTests.cs]: SearchQuery = string returned as selected by the accessible-autocomplete component

**Review by CathLass — APPROVED** [review]

**Pull request merged**